site/docs/user-manual.html

```diff
@@ -2681,6 +2681,15 @@ <h3 id='canonicalize'><code>canonicalize-flags</code></h3>
   Note that a small number of options cannot be reordered, because Bazel cannot
   ensure that the effect is identical.
 </p>
+<p>
+  As an example:
+</p>
+<pre>
+  % bazel canonicalize-flags -- --config=any_name --test_tag_filters="-lint"
```

Comment:
It's worth explicitly calling out that the command doesn't expand `config` flags, even though you have an example here already.

Reply:
Done. Thanks!